Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN chmod a+rwX /files
 RUN set -x \
 #  && apt-get update \
   && apt-get install -y inotify-tools \
-  && wget -O /files/filebot.deb 'https://app.filebot.net/download.php?type=deb&arch=amd64&version=4.7.2' \
+  && wget -O /files/filebot.deb 'https://app.filebot.net/download.php?type=deb&arch=amd64&version=4.7.7' \
```

Comment:
I'd add the `SHA-256` and check that the binary you get is indeed the binary you'd expect.